File: examples/hello.cpp
#include <iostream>

std::string hello() {
    return "Hello, World!";
}

void say_hey(std::string name) {
    std::cout << "Hey, " << name << "!" << std::endl;
}

int main() {
    std::cout << hello() << std::endl;
    std::string name;
    std::cout << "Enter your name: ";
    std::cin >> name;
    say_hey(name);

    return 0;
}

File: examples/hello.py
def hello() -> str:
  return "Hello, World!"

def say_hey(name: str) -> None:
  print("Hey, ", name, "!", '\n', sep='', end='')

def main() -> int:
  print(hello(), '\n', sep='', end='')
  print("Enter your name: ", end='')
  name = str(input())
  say_hey(name)
  return 0

exit(main())
